omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,418 @@
+Group and Data Organizer is a role of someone tasked with creation of new
+users and groups in OMERO and allocating the users to appropriate groups
+(also able to change the users' information such as e-mail and change
+group permissions level). These tasks are facilitated by the privileges
+:term:`Create and Edit Groups`,
+:term:`Create and Edit Users` and :term:`Add Users to Groups`.
+
+Group and Data Organizer might also be tasked with dealing with data left
+after OMERO users which left the institution. Organizer can transfer of
+ownership of the data of the leaving person (facilitated by :term:`Chown`
+privilege) to another user. Possibly, the new owner of the data is not a
+member of the data group. In such case, Organizer first moves the data
+between groups (facilitated by :term:`Chgrp` privilege), only then
+transfers the ownership of the data. Always try to avoid the situation
+where owner of the data is not in data group.
+
+For moving data between groups, usage of OMERO.web is highly recommended.
+Organizer can create new containers (Projects, Datasets) on behalf of data
+owner in OMERO.web conveniently as part of Move to Group command in OMERO.
+web (link to Move to Group in Help). The containers and links of data to
+containers will belong to data owner. For new container creation and
+linking, :term:`Write Data` privilege is necessary.
+
+In case of data owner not being in the group where the data is,
+Organizer can also add the new data owner to the
+data group (facilitated by :term:`Add Users to Groups` privilege),
+instead of moving the data. Organizer transfers the ownership of the data
+to the new owner only after he/she has added the new
+data owner to the data group.
+
+During all data manipulation steps, Organizer needs :term:`Write Data`
+privilege to create new Projects or Datasets for the new owners of the
+data and make the linking of the data to those containers or to already
+existing containers of the new owner. Note that any links created by
+Organizer will belong to the Organizer, not the owner of the data. This
+will be addressed in OMERO.web in the 5.3.x series. The ownership transfer
+of the containers and links can be done later on the CLI. Linking of
+others' data is never possible in Private groups.
+
+After the Organizer dealt with the data, he/she can remove the leaving
+person from any group (included in :term:`Add Users to Groups` privilege)
+and make the user inactive (facilitated by :term:`Create and Edit Users`
+privilege).
+
+Note that the ownership of data of a user can be trasferred either
+piecemeal, i.e. specifying each Project or Dataset to transfer (using
+``omero chown`` command of CLI), or all data of the user can be
+transferred in one step (this function is availabel in API and will appear
+in OMERO clients in the 5.3.x series of OMERO).
+
+Quite naturally the Group and Data Organizer can be easily split into two
+separate roles, Group Organizer (with :term:`Create and Edit Groups`,
```

Comment:
with the Group Organiser having xxx privileges, and the Data Organiser having yyy privileges